packages/react-dom/src/events/getEventCode.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code#Code_values
+ * @param {object} nativeEvent
+ * @returns {string} Standardized keyboard code value if defined natively
```

Comment:
I think you can remove the JSDocs type annotations since we use Flow anyway.

This file should also add the usual copyright header and a flow hint:

https://github.com/facebook/react/blob/679402a66b09e129d06f415cd976d18727cc8590/packages/create-subscription/src/createSubscription.js#L1-L8